mutation-testing/shell-scripts/create-stable.sh

```diff
@@ -0,0 +1,55 @@
+# for specific packages creates the outpup
+
+# removes everything except .txt files
+
+#!/bin/bash
+
+# moves to mutation-testing folder
+cd ../packages-output
```

Comment:
Shellcheck warning:
```
cd ../packages-output
^-------------------^ SC2164 (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```